gameoflife.php

```diff
@@ -36,7 +51,7 @@
 
 if ($options->getOption("version"))
 {
-    echo "Game of Life\nVersion 1.0, CN-Consult GmbH 2019-2019\n";
+    echo "Game of Life\nVersion 2.0, CN-Consult GmbH 2019-2019\n";
```

Comment:
Haha, yes, that deserves a 2.0 version! 😄 
But 2019-2019 does not make so much sense. Usually one starts with the current year only, and if you do changes in the next year, you can change it to 2019-2020.